src/web/DiscreteGestureHandler.ts

```diff
@@ -32,7 +32,7 @@ class DiscreteGestureHandler extends GestureHandler {
     );
   }
 
-  transformNativeEvent({ center: { x, y } }) {
+  transformNativeEvent({ center: { x, y } }: any) {
```

Comment:
`any`.

Reply:
These web files are now marked with a `no-check`, so I will just leave these `any` to be removed when we type it.